### DISCUSSION TITLE
I'm encountering an error when trying to convert a CSV file to Excel using Python Mojo

### DISCUSSION THREAD

**Discussion by mohanreddy7892** [modular/modular#902] [Q&A]
```
data.csv 
Name, Age, Occupation, Country
John Doe, 28, Engineer, USA
Jane Smith, 34, Designer, UK
Alice Johnson, 29, Artist, Australia
Bob Brown, 22, Student, Canada
Charlie White, 31, Doctor, New Zealand
Diana Green, 40, Lawyer, India
Eve Black, 25, Photographer, France
Frank Gray, 37, Journalist, Germany
Grace Blue, 29, Architect, Italy
Henry Pink, 45, Chef, Spain
```

```
from python import Python
fn main() raises:
    let pd = Python.import_module("pandas")
    let csv_file = "/home/mohan/data.csv"
    let excel_file = "/home/mohan/data.xlsx"
    let df = pd.read_csv(csv_file)
    print(df)
    df.to_excel(excel_file, index=False, engine="openpyxl")
    print("CSV has been converted to Excel!")
```
```
**But im getting Error :**
Unhandled exception caught during execution: An error occurred in Python.
mojo: error: execution exited with a non-zero result: 1
```

**Comment by gujuriu** [comment]
Hi @mohanreddy7892, I tried your example and for me it works ~fine, it reads and prints the content of .csv file but the xlsx file is not written to the path mentioned in my case "./data1.xlsx", I don't know yet why.
Can it be that you installed pandas only in a virtual environment and you did not exported path to MOJO_PYTHON_LIBRARY? If yes, you might find a work around [here](https://github.com/modularml/mojo/issues/854).
So, basically, in my case pandas was installed in the python installation that mojo comes with (3.10.12), but also works on a conda env if you use python 3.10 or lower, does not work with python 3.11.x (will throw some errors and segmentation fault).

**Reply by rd4com** [reply]
@mohanreddy7892, @gujuriu yes i ran into the same issue when testing matplotlib, as gujuriu said, i needed to install matplotlib using pip in the python env used by mojo.
